_docs/layout/navigation.md

```diff
@@ -135,65 +149,70 @@ for highly compliant Helix applications is the [left navigation](#side-nav).
 {% endcolumn %}
 </div>
 
-### Top Navigation Spacing
+### Top navigation spacing
 
 <div class="row">
 {% column left:"col-md-4" %}
--   Horizontal spacing includes 20px left margin
 
--   Each menu item button has 20px left and right padding
+-   Horizontal spacing includes 20px of left margin.
+
+-   Each menu item button has 20px of left and right padding.
 
-    {% include note.html content="Dropdown padding left begins at the left
-    border of the parent primary navigational item and padding right begins
-    after the longest textual element in the dropdown element." %}
+    {% include note.html content="Drop-down element left padding
```

Comment:
"dropdown" is not a noun; however, "drop-down" describes a "thing" such as element or menu. Also, I'd prefer using the simpler `navigation` item versus `navigational` item.
Revised to read:
```
The left padding of a drop-down element begins at the left border 
of the parent primary navigation item and the right padding begins 
after the longest text element in the drop-down element.
```